python/tests/plugins/test_gen_json.py

```diff
@@ -10,6 +11,9 @@
 
 
 class TestGenJson(TestCase):
+    def setUp(self) -> None:
```

Comment:
If this `setUp` function is going to be needed in all our tests, we may benefit from having a `BaseTestCase` class that handles that so we don't forget. But I'm not sure if we actually need that in every test class.

Reply:
This is true only for testing plugins and more E2E scenarios. We can leverage creating our own LanguageContext for a number of the unit tests, so I'll make an issue to create a common test class for tests that involve the active context.